packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -205,44 +354,52 @@ def reduce_logging():
 
 @pytest.mark.skipif(not lb_enabled(), reason='Load Balancer disabled')
 def test_vip(dcos_api_session, reduce_logging):
-    '''Test VIPs between the following source and destination configurations:
-        * containers: DOCKER, UCR and NONE
-        * networks: USER, BRIDGE (docker only), HOST
-        * agents: source and destnations on same agent or different agents
-        * vips: named and unnamed vip
-
+    '''Test every permutation of VIP
     '''
-    addrs = [['1.1.1.{}:{}', '1.1.1.{}:{}'],
-             ['/namedvip{}:{}', 'namedvip{}.marathon.l4lb.thisdcos.directory:{}']]
     # tests
     # UCR doesn't support BRIDGE mode
-    permutations = [[c, vi, va, sh, vn, pn]
+    permutations = [[c, ss, sh, vn, pn]
```

Comment:
@aeyakovenko I think you hit the nail on the head. I don't think its particular to python, but I prefer that if the usage of the variable is pretty trivial (here, you are just constructing an argument list) and it should definitely NOT be used outside the scope of some loop (like here) then its good to use minimal names. The second you see a name like these you know about the limited usage.

That being said, in python, the general convention is all variables should be very descriptive, so if you only have to write it twice anyway, it is really not to much to ask that you write it out and then the very complicated constructor becomes way more intelligible.

When reviewing this, I had to copy paste VipTest next to the constructor just to figure out what `ss` is because  it certainly doesnt seem to match the constructor argument `is_named`. The same goes for most of those names. So I would ask that you rename these to be perfectly in sync with the args they represent.

Given that you should rename them and that @asridharan already found it cryptic, I think you should definitely write it out here, but in future cases when you only have one or two iterators, feel free to use this form